deepchem/models/torch_models/tests/test_molformer.py

```diff
@@ -113,3 +113,21 @@ def test_molformer_save_reload(tmpdir):
 
     # all keys values should match
     assert all(matches)
+
+
+@pytest.mark.hf
+def test_molformer_finetuning_multitask_classification():
```

Comment:
Please add a unit test for multi-task regression finetuning as well

Reply:
Done